index.html

```diff
@@ -79,7 +90,7 @@ <h1>I'm Gilbert,<br> Glad to See <br>You!</h1>
 
       <div class="Card-Works">
```

Comment:
- [ ] Good job on your design 👍🏽 . Your project cards contain the project's picture and the project's description, which should switch places with each project (picture is on the left and description on the right for the first project, description on the left and picture on the right for the second project and so on and so forth).

| Figma Design  | Your Design |
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/63473584/199578710-e0f8729e-4619-414e-957d-092413f38e83.png)  | ![image](https://user-images.githubusercontent.com/63473584/199578851-83336ff7-77bf-4c62-a106-6d258b79f8fb.png)  |